libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -357,8 +376,8 @@ export async function fetchAndProcessPlainHtml({ url, shouldDecorateLinks = true
 
   const blocks = body.querySelectorAll('.martech-metadata');
   if (blocks.length) {
-    import('../../martech-metadata/martech-metadata.js')
-      .then(({ default: decorate }) => blocks.forEach((block) => decorate(block)));
+    const { default: decorate } = await import('../../martech-metadata/martech-metadata.js');
```

Comment:
Does this need to be blocking if before it wasnt?

Reply:
With this non blocking behaviour it is printing error for clientid 'milo'.
<img width="1116" alt="Screenshot 2024-06-07 at 10 14 31 AM" src="https://github.com/adobecom/milo/assets/41122199/727ceb54-1283-43c9-9c21-148bcff59c6d">

But we need this log to be logged with clientId as 'feds-milo'. So fixing it by adding log with `catch` as errorType `warn`. Like shown below
 
<img width="1346" alt="Screenshot 2024-06-07 at 10 17 01 AM" src="https://github.com/adobecom/milo/assets/41122199/bc77ffaf-f8fb-48d5-bc77-0b52017adad0">